doc/INSTALL.md

```diff
@@ -43,6 +43,11 @@ Get dependencies:
     pip3 install --user poetry
     poetry install
 
+If you are getting errors that `/home/<your username>/.local/bin/` is not in `PATH`, you will need to add it.
+
+    echo -e "\n\n"'export PATH="/home/<your username>/.local/bin:$PATH"' >>~/.bashrc
+    source ~/.bashrc
+
```

Comment:
nit: I think this is information not related to c-lightning install procedure, and this should not stay in the docs IMHO.

Reply:
I put that in there because I was getting that error with a fresh install of Ubuntu. I was thinking of users that may want to run a node but that aren't as familiar with linux. Now that I'm looking at the main README.md again, I see that people are expected to install with a ppa, so maybe I shouldn't have worried about that little part.

I guess I was thinking of myself early on and how many times I wish there had been something like that in docs that I was reading. I guess they can just type their error message into a search engine and figure it out.

Reply:
>I guess they can just type their error message into a search engine and figure it out.

Yes, usually I type the error on google before read 20 lines of docs

Reply:
You actually only need to log in again: if pip created the dir, it wasn't in your path.